### PR TITLE
write calculateRoll() method

### PR DESCRIPTION
**Pull request by rileydlynch**

### CODE REVIEW

File: DnDDieRollCalculator/Program.cs [hunk 1]
```diff
@@ -1,2 +1,21 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+namespace RollReturn
+{
+    class Program
+    {
+        static int CalculateRoll(int dievalue)
```
Comment (fromjaytoz) [inline DnDDieRollCalculator/Program.cs]: change dievalue to dieValue

File: DnDDieRollCalculator/Program.cs [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+        {
+
+            Random random = new Random();
+            int Roll = 0;
```
Comment (fromjaytoz) [inline DnDDieRollCalculator/Program.cs]: change Roll to roll

**Review by fromjaytoz — CHANGES_REQUESTED** [review]

File: DnDDieRollCalculator/Program.cs [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+
+            Roll += random.Next(1, dievalue);
```
Comment (fromjaytoz) [inline DnDDieRollCalculator/Program.cs]: also, why not do:
int Roll = random.Next(1, dieValue);

File: DnDDieRollCalculator/Program.cs (hunk 2 above)
Comment (rileydlynch) [reply DnDDieRollCalculator/Program.cs]: Changed to "roll"

File: DnDDieRollCalculator/Program.cs (hunk 1 above)
Comment (rileydlynch) [reply DnDDieRollCalculator/Program.cs]: Changed to "dieValue"

File: DnDDieRollCalculator/Program.cs [hunk 4]
```diff
@@ -1,2 +1,21 @@
...
+    {
+        static int CalculateRoll(int dieValue)
+        {
+
+            Random random = new Random();
+            int roll = 0;
+
+            roll += random.Next(1, dieValue);
+
+            return roll;
```
Comment (fromjaytoz) [inline DnDDieRollCalculator/Program.cs:14]: honestly, we're inevitably returning the value of the package's random.next(), which makes this function redundant.

Comment (fromjaytoz) [reply DnDDieRollCalculator/Program.cs:14]: aka, wherever you intended to use CalculateRoll(), you can just call random.Next() directly

**Review by fromjaytoz — CHANGES_REQUESTED** [review]

File: DnDDieRollCalculator/Program.cs (hunk 3 above)
Comment (fromjaytoz) [reply DnDDieRollCalculator/Program.cs]: also, it should be `int roll`, not ` int Roll`

Comment (rileydlynch) [reply DnDDieRollCalculator/Program.cs]: image

The capitalization has already been fixed. And you can't assign a return value to a variable without initializing it first, as the image shows.

**Pull request closed without merging**